pkg/sriov/list_test.go

```diff
@@ -0,0 +1,349 @@
+package sriov
+
+import (
+	"fmt"
+	"testing"
+
+	srIovV1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestNetworkList(t *testing.T) {
+	testCases := []struct {
+		testNetwork   []*NetworkBuilder
+		nsName        string
+		listOptions   []v1.ListOptions
+		expectedError error
+		client        bool
+	}{
+		{
+			testNetwork:   []*NetworkBuilder{buildValidSriovNetworkTestBuilder(buildTestClientWithDummyObject())},
+			nsName:        defaultNetNsName,
```

Comment:
IMO, using variable defined in other files makes it hard to understand what you are passing into the test. I think I prefer just using hard coded strings for the test inputs.

Reply:
I agree.  I was actually looking for the definition of these vars but couldn't find them 😄